openshift_pipelines/understanding-openshift-pipelines.adoc

```diff
@@ -0,0 +1,28 @@
+[id="understanding-openshift-pipelines"]
+= Understanding OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: understanding-openshift-pipelines
+toc::[]
+
+:FeatureName: OpenShift Pipelines
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. It uses link:https://tekton.dev[Tekton] building blocks to automate deployments across multiple platforms by abstracting away the underlying implementation details. Tekton introduces a number of standard link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)] for defining CI/CD pipelines that are portable across Kubernetes distributions.
+
+[id="op-key-features"]
+== Key features
+
+* {pipelines-title} is a serverless CI/CD system that runs Pipelines with all the required dependencies in isolated containers.
+* {pipelines-title} are designed for decentralized teams that work on microservice based architecture.
+* {pipelines-title} use standard CI/CD pipeline definitions that are easy to extend and integrate with the existing Kubernetes tools, enabling you to scale on-demand.
+* You can use {pipelines-title} to build images with Kubernetes tools such as Source-to-Image (S2I), Buildah, Buildpacks, and Kaniko that are portable across any Kubernetes platform.
+* You can use the {product-title} Developer Console to create Tekton resources, view logs of Pipeline runs, and manage pipelines in your {product-title} namespaces.
+
+include::modules/op-pipelines-concepts.adoc[leveloffset=+1]
+
+== Additional resources
```

Comment:
Please remove links to GitHub repos or request approval from Eric Rich.

Reply:
I have removed one link. For rest of the  link in this doc, I have received approval from the @RickJWagner. I will forward you the email with list of links for which the approval is received.